src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -16,8 +16,7 @@ namespace System.Diagnostics.Tests
 {
     public partial class ProcessTests : ProcessTestBase
     {
-        private const string s_xdg_open = "xdg-open";
-        private const int s_exit_code_kill = 137;  // using exit code 137 to show the process was killed
+        private const int EXIT_CODE_KILL = 137;  // using exit code 137 to show the process was killed
```

Comment:
per https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md#c-coding-style

it should be written `private const string ExitCodeKill = 137`